packages/mediacenter/kodi/scripts/pastekodi

```diff
@@ -92,6 +102,10 @@ fi
   cat_file "${KODI_ROOT}/.smb/user.conf"
   cat_file "/run/samba/smb.conf"
 
+  ls_dir /storage/.config
```

Comment:
why not just do this recursive?

Reply:
because I wanted to keep the output limited to the most important stuff.

There's too much stuff in .config that's not really interesting - pulse temp files, aacs keys, rpi4 bootload backup files etc - our .config dir has become a hybrid of /etc and /var ....

Reply:
It just seems like it's going to be wack-a-mole. What about `modules-load.d` what about `firmware` (custom), etc? Not a big deal but will make it so we don't have to keep adding in the future

Reply:
modules-load.d and firmware are likely to show up in journal as modules/firmware entries.

I also didn't include RPi firmware version as that info is also available via dmesg/journal.

pastekodi should provide the most important info, not clutter with too many seldomly needed stuff. For everything else we can ask users for specific info.